app/models/program.rb

```diff
@@ -199,4 +204,33 @@ def check_languages_format
     # We check if every language is a valid ISO 639-1 language
     errors.add(:languages, 'must be ISO 639-1 valid codes') unless languages_array.select{ |x| ISO_639.find(x).nil? }.empty?
   end
+
+  ##
+  # Check if schedule_interval is a divisor of 60 minutes
+  #
+  def schedule_interval_divisor_60
+    errors.add(:schedule_interval, 'must be a divisor of 60') if schedule_interval > 0 && 60 % schedule_interval > 0
+  end
+
+  ##
+  # Unschedule all the events which don't fit
+  #
+  def unschedule_unfit_events
+    conference_start_hour = conference.start_hour
+    unfit_schedules = event_schedules.select do |event_schedule|
```

Comment:
as the conference always started in 00 and the `schedule_interval` is a divisor of 60, this can be written simpler: :wink: 

```
unfit_schedules = event_schedules.select do |event_schedule|
event_schedule.start_time.min % schedule_interval
end
```